imports/client/hooks/useCallState.ts

```diff
@@ -355,9 +380,7 @@ const useTransport = (
       return undefined;
     }
   }, [
-    device, hasParams, transportParams?._id, direction,
-    transportParams?.transportId, transportParams?.iceParameters, transportParams?.iceCandidates,
-    transportParams?.dtlsParameters, dispatch,
+    device, hasParams, transportParams, direction, dispatch,
```

Comment:
Did you intend to switch semantics to "whenever the object identity changes in any way, rerun this effect?" here?  Because as-written this is going to cause us to recreate both the `recv` and `send` transports when either changes because the `useTracker` block above comingles the two.

We should probably rework that chunk to make the specific results preserve identity slightly better, but I do think only updating here when the specific fields we care about change is the preferable set of semantics

Reply:
Ah, that's a good point. I just got annoyed at how long the list was and thought it seemed unnecessary. I'll add back the full list.